demos/get-events.ts

```diff
@@ -2,8 +2,8 @@ import "websocket-polyfill";
 import NDK, { NDKRelay } from "@nostr-dev-kit/ndk";
 import { relaysFromArgs } from "./utils/relays-from-args";
 
-import chalk from 'chalk';
-import { verifySignature } from "nostr-tools";
+import chalk from "chalk";
+import { verifyEvent } from "nostr-tools";
```

Comment:
import from `nostr-tools/pure` instead

Reply:
I've actually reimplemented the verifySignature stuff in NDK (but we haven't cut a release since) so this will change to the in-house version as soon as we have a new release out.